app/ante/gas_wanted_test.go

```diff
@@ -83,7 +83,7 @@ func (s *AnteTestSuite) TestGasWantedDecorator() {
 	for _, tc := range testCases {
 		s.Run(tc.name, func() {
 			deps := evmtest.NewTestDeps()
-			stateDB := deps.StateDB()
+			stateDB := deps.NewStateDB()
```

Comment:
The function is renamed from `StateDB` to `NewStateDB` everywhere to reflect that fact that it's not a getter for an existing `StateDB` and instead generates and returns a new pointer for a `StateDB` with a fresh journal